### PR TITLE
Enable incentives to be removed by admins

### PR DESCRIPTION
**Pull request by Wrenderer**
* Add `destroy` action to incentives controller
 * Fix bug - forgetting a description when creating or editing incentives would just crash; now it just gives a clean error message

[Finished: #101315088]
[Finished: #101623970]

### CODE REVIEW

File: app/controllers/incentives_controller.rb [hunk 1]
```diff
@@ -44,10 +44,24 @@ def update
                   notice: "Incentive was successfully updated.",
                   only: true
```
Comment (michaelwehrley) [inline app/controllers/incentives_controller.rb]: @Wrenderer What does `only: true` mean? I can't find it referenced anywhere.  Thanks!

File: app/controllers/incentives_controller.rb [hunk 2]
```diff
@@ -44,10 +44,24 @@ def update
...
     else
+      populate_image_choices
       render :edit
     end
   end
 
+  def destroy
```
Comment (ericcf) [inline app/controllers/incentives_controller.rb]: I'd like to see unit tests for this controller action.

File: app/controllers/incentives_controller.rb (hunk 1 above)
Comment (Wrenderer) [inline app/controllers/incentives_controller.rb]: Actually not sure; the redirects were carried over from those lines that were already there (43-45), so if we're sure it isn't doing anything we can remove it in all places. Maybe @eschlange knows?

Comment (eschlange) [inline app/controllers/incentives_controller.rb]: Not sure about this, I can't find a reference to this in any documentation. If I wrote it, it should be removed.

Comment (Wrenderer) [inline app/controllers/incentives_controller.rb]: Alright, removing all `only:true` uses then, thanks.

**Comment by Wrenderer** [comment]
Tests added

**Comment by ericcf** [comment]
Thanks!

**Pull request merged**